pkg/collector/corechecks/cluster/orchestrator/collectors/k8s/namespace.go

```diff
@@ -0,0 +1,100 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+//go:build kubeapiserver && orchestrator
+// +build kubeapiserver,orchestrator
+
+package k8s
+
+import (
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/collectors"
+	"github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/processors"
+	k8sProcessors "github.com/DataDog/datadog-agent/pkg/collector/corechecks/cluster/orchestrator/processors/k8s"
+	"github.com/DataDog/datadog-agent/pkg/orchestrator"
+	corev1Informers "k8s.io/client-go/informers/core/v1"
+	corev1Listers "k8s.io/client-go/listers/core/v1"
+
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/tools/cache"
+)
+
+// NewNamespaceCollectorVersions builds the group of collector versions.
+func NewNamespaceCollectorVersions() collectors.CollectorVersions {
+	return collectors.NewCollectorVersions(
+		NewNamespaceCollector(),
+	)
+}
+
+// NamespaceCollector is a collector for Kubernetes Namespaces.
+type NamespaceCollector struct {
+	informer  corev1Informers.NamespaceInformer
+	lister    corev1Listers.NamespaceLister
+	metadata  *collectors.CollectorMetadata
+	processor *processors.Processor
+}
+
+// NewNamespaceCollector creates a new collector for the Kubernetes
+// Namespace resource.
+func NewNamespaceCollector() *NamespaceCollector {
+	return &NamespaceCollector{
+		metadata: &collectors.CollectorMetadata{
+			IsDefaultVersion: true,
+			IsStable:         true,
```

Comment:
This should be false at start.

Reply:
Ah - that's the flag for enabled by default - thanks